web/viewer.css

```diff
@@ -494,34 +468,29 @@ html[dir='ltr'] .doorHangerRight:before {
   background-color: rgb(255, 102, 102);
 }
 
-html[dir='ltr'] #toolbarViewerLeft {
-  margin-left: -1px;
```

Comment:
Yes. The one pixel difference didn't make sense because it caused a difference between the leftmost and rightmost button (albeit minor). Removing the rules not only simplifies the CSS but also makes the margin of the leftmost button equal to the margin of the rightmost button.